hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestOzoneClientMultipartUploadWithFSO.java

```diff
@@ -311,6 +316,124 @@ public void testUploadPartOverrideWithRatis() throws IOException {
         commitUploadPartInfo.getPartName());
   }
 
+  @Test
+  public void testUploadTwiceWithEC() throws IOException {
+
+    String volumeName = UUID.randomUUID().toString();
```

Comment:
Nearly every test in that class starts with the same repeated code to create a volName, create the volume, create the bucketname, bucket etc. Could you refactor it into the setup method for all tests to reduce the repetition?

Reply:
Updated...
@sodonnel